WooCommerce/WooCommerceTests/ViewRelated/Coupons/CouponDetailsViewModelTests.swift

```diff
@@ -0,0 +1,20 @@
+import XCTest
+@testable import Yosemite
+@testable import WooCommerce
+
+final class CouponDetailsViewModelTests: XCTestCase {
```

Comment:
nit: might be nice adding a test case for coupons of `percent` discount type, especially on the `amount` 😄

Reply:
Added the test in 75d6618983194ab6b2e102b7e85cb7e2d40af403.